packages/create-lz-oapp/src/utilities/installation.ts

```diff
@@ -1,9 +1,22 @@
 import type { Config, PackageManager } from '@/types'
+import { createModuleLogger } from '@layerzerolabs/io-devtools'
 import { spawn } from 'child_process'
 import which from 'which'
 
 export const installDependencies = (config: Config) =>
     new Promise<void>((resolve, reject) => {
+        const logger = createModuleLogger('installation')
+
+        // We'll store combined stdout and stderr in this variable
+        const std: string[] = []
+
+        // This function will handle stdout/stderr streams from the child process
+        const handleStd = (chunk: string) => {
+            std.push(chunk)
```

Comment:
Could cause overflow, but unlikely in this case.

Reply:
Yeah we roughly know how much output these produce so that should be fine